docs/ops/pooling/MaxPool_1.md

```diff
@@ -52,15 +52,16 @@
     * *floor*
   * **Type**: string
   * **Default value**: *floor*
+  * **Required**: *no*
 
 * *auto_pad*
 
   * **Description**: *auto_pad* how the padding is calculated. Possible values:
-    * None (not specified): use explicit padding values.
+    * explicit: use explicit padding values.
     * *same_upper (same_lower)* the input is padded to match the output size. In case of odd padding value an extra padding is added at the end (at the beginning).
     * *valid* - do not use padding.
   * **Type**: string
-  * **Default value**: None
+  * **Default value**: *explicit*
   * **Required**: *no*
```

Comment:
@lazarevevgeny If MO always generates this attribute, does it mean that it is required?
Or in general required means that we can work without this attribute

Reply:
Nope. See the comment above